.moai/specs/SPEC-AGENT-MODEL-ENFORCE-001/acceptance.md

```
@@ -0,0 +1,318 @@
+
+**AC-AME-035**: 게이트 OFF 경로에 추가 파일 I/O가 없다.
+```bash
+go test ./internal/hook/ -run TestAgentModelGuardDisabledNoExtraIO -v
+# 테스트 본문: 게이트 OFF에서 Handle 실행 후 t.TempDir() 하위에 신규 파일이
+#   감사 로그 1개 외에는 생성되지 않음을 단언 (설정 재로드 등 부수 I/O 부재)
+```
+
+### §D.5 M5 — 규칙 가시성
+
+**AC-AME-040**: per-spawn 모델 주입 의무를 담은 조각이 항상 로드 표면에 있다.
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)